client/components/App.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+import axios from 'axios';
+import styled from 'styled-components';
+import GlobalStyles from '../globalStyles.js';
+import Search from './Search.jsx';
+import Votes from './Votes.jsx';
+import QuestionsAnswers from './QuestionsAnswers.jsx';
+
+class CustomerQuestions extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+
+    }
+  }
+
+  render() {
+    return (
+      <div>
+        <GlobalStyles/>
+        <h2>Customer questions & answers</h2>
+        <div>
+          <Search/>
+        </div>
+        <AskContainer>
+          <Votes/>
+          <QuestionsAnswers/>
+        </AskContainer>
+      </div>
+    )
+  }
+}
+
+export default CustomerQuestions;
+
+// Styles
+const AskContainer = styled.div`
```

Comment:
On styled-components site, syles were defined before render. You have defined it outside. Is there any specific reason for this approach ?

Reply:
Good question. I saw it being suggested on a blog and thought it was a good idea. However since the docs define styles before the component, I will move it back.

Reply:
If its suggested in a blog, lets check with TM. Would be good to know their thoughts.